oap-server/exporter/src/main/java/org/apache/skywalking/oap/server/exporter/provider/grpc/GRPCExporter.java

```diff
@@ -56,9 +56,15 @@ public GRPCExporter(GRPCExporterSetting setting) {
         subscriptionSet = new HashSet<>();
     }
 
-    @Override public void export(MetricsMetaInfo meta, Metrics metrics) {
-        if (subscriptionSet.size() == 0 || subscriptionSet.contains(meta.getMetricsName())) {
-            exportBuffer.produce(new ExportData(meta, metrics));
+    @Override public void export(ExportEvent event) {
+        if (ExportEvent.EventType.TOTAL.equals(event.getType())) {
```

Comment:
`EventType` is enum type so we can just replace `.equals` with `==`? As you said in another comment (I don't exactly remember which one), it reduce a unnecessary call stack frame

Reply:
I said this for `equal` PR, not merged. For enum, `==` is OK :)